### PR TITLE
fix: compare the returned ids when deleting subscriptions

### PR DESCRIPTION
**Pull request by abc3**

### CODE REVIEW

File: lib/realtime/application.ex [hunk 1]
```diff
@@ -75,8 +75,8 @@ defmodule Realtime.Application do
         RealtimeWeb.Endpoint,
         RealtimeWeb.Presence,
         {Task.Supervisor, name: Realtime.TaskSupervisor},
-        Realtime.Latency,
-        Realtime.Telemetry.Logger
+        Realtime.Latency
+        # Realtime.Telemetry.Logger
```
Comment (abc3) [inline lib/realtime/application.ex:79]:
```suggestion
        Realtime.Telemetry.Logger
```

File: lib/realtime/application.ex [hunk 2]
```diff
@@ -75,8 +75,8 @@ defmodule Realtime.Application do
         RealtimeWeb.Endpoint,
         RealtimeWeb.Presence,
         {Task.Supervisor, name: Realtime.TaskSupervisor},
-        Realtime.Latency,
-        Realtime.Telemetry.Logger
+        Realtime.Latency
```
Comment (abc3) [inline lib/realtime/application.ex:78]:
```suggestion
        Realtime.Latency,
```

**Review by chasers — APPROVED** [review]

File: lib/extensions/postgres_cdc_rls/subscription_manager.ex [hunk 3]
```diff
@@ -160,11 +162,22 @@ defmodule Extensions.PostgresCdcRls.SubscriptionManager do
...
+              remaining_ids ->
+                Logger.warn(
+                  "not all subscriptions were deleted from the queue: #{inspect(length(remaining_ids))}"
+                )
+
+                :queue.from_list(remaining_ids) |> :queue.join(q1)
+            end
+
+          other ->
+            Logger.error("delete subscriptions from the queue failed: #{inspect(other)}")
```
Comment (chasers) [inline lib/extensions/postgres_cdc_rls/subscription_manager.ex:180]: `delete` make `Delete`

File: lib/extensions/postgres_cdc_rls/subscription_manager.ex [hunk 4]
```diff
@@ -160,11 +162,22 @@ defmodule Extensions.PostgresCdcRls.SubscriptionManager do
...
         case Subscriptions.delete_multi(state.conn, ids) do
-          {:ok, _} ->
-            q1
-
-          {:error, reason} ->
-            Logger.error("delete subscriptions from the queue failed: #{inspect(reason)}")
+          {:ok, %{rows: [_ | _] = rows}} ->
+            case ids -- List.flatten(rows) do
+              [] ->
+                Logger.debug("delete subscriptions from the queue success")
```
Comment (chasers) [inline lib/extensions/postgres_cdc_rls/subscription_manager.ex:168]: `Delete`

File: lib/extensions/postgres_cdc_rls/subscription_manager.ex [hunk 5]
```diff
@@ -160,11 +162,22 @@ defmodule Extensions.PostgresCdcRls.SubscriptionManager do
...
+                q1
+
+              remaining_ids ->
+                Logger.warn(
+                  "not all subscriptions were deleted from the queue: #{inspect(length(remaining_ids))}"
```
[the remark this answers is not in the document]
Comment (abc3) [reply lib/extensions/postgres_cdc_rls/subscription_manager.ex:173]: I guess `length(rows)`?

**Pull request closed without merging**